apertium-spa.spa.dix

```diff
@@ -87378,6 +87378,9 @@
 <e lm="íncubo" a="albertonl">   <i>íncubo</i><par n="abismo__n"/></e>
 <e lm="ñapa" a="albertonl">   <i>ñapa</i><par n="abeja__n"/></e>
 <e lm="óbolo" a="albertonl">   <i>óbolo</i><par n="abismo__n"/></e>
+<e lm="ese" a="albertonl">  <i>ese</i><par n="abeja__n"/></e>
```

Comment:
"Ese" as a letter name is OK. But as it is a very common determiner or pronoun ("ese coche", "con ese"), some disambiguation rules will be necessary to make sure it is properly analyzed. "Con ese" is ambiguous and context is needed to disambiguate.